docker/config/l1-node/Dockerfile

```diff
@@ -1,16 +1,5 @@
-FROM golang:1.23-alpine as builder
+FROM ethpandaops/ethereum-genesis-generator:pk910-bash-el-genesis-generator
 
-RUN apk add --no-cache git && git clone https://github.com/protolambda/eth2-testnet-genesis.git \
-    && cd eth2-testnet-genesis \
-    && go install .
+COPY ./generate-genesis.sh /config/generate-genesis.sh
 
-COPY ./generate-genesis.sh /generate-genesis.sh
-
-FROM alpine:3.20.3
-
-RUN  apk add --no-cache bash
-
-COPY --from=builder /generate-genesis.sh /config/generate-genesis.sh
-COPY --from=builder /go/bin/eth2-testnet-genesis /usr/local/bin/
-
-ENTRYPOINT [ "/bin/sh" , "/config/generate-genesis.sh" ]
+ENTRYPOINT [ "/bin/bash", "/config/generate-genesis.sh" ]
```

Comment:
The build time is much faster than that from the previous dockerfile: e.g. 12.7sec vs 38.2sec